db/merge_helper.cc

```diff
@@ -140,15 +140,17 @@ Status MergeHelper::MergeUntil(InternalIterator* iter,
   ParsedInternalKey orig_ikey;
   bool succ = ParseInternalKey(original_key, &orig_ikey);
   assert(succ);
+  Status s;
```

Comment:
Are these changes necessary?

Reply:
It gives error because Status s is defined but its status is not checked in the flow where direct status is returned  instead of s. So to make it consistent I assigned the error messages to s and returned s.